org.opent2t.sample.hub.superpopular/com.insteon.hub/js/thingTranslator.js

```diff
@@ -340,17 +342,29 @@ class Translator {
      * We can identify the device using those two value based on the table at
      * https://insteon.atlassian.net/wiki/display/IKB/Insteon+Device+Categories+and+Sub-Categories#InsteonDeviceCategoriesandSub-Categories-devcat-subcat
      */
-    _getOpent2tInfo(devCat, subCat) {
+    _getOpent2tInfo(deviceData) {
+        var devCat = deviceData.DevCat
+        var subCat = deviceData.SubCat.toString(16).toUpperCase();
         switch (devCat) {
+            case 0:
+                if( subCat === '0' && deviceData.ProductType ==='thermostat'){
+                    return undefined;       //Nest thermostat, not supported now
+                }
+                return undefined;
             case 1:
                 if (this._subCatMap.lightBulbs.indexOf(subCat) >= 0)
                 {
                     return {
                         "schema": 'org.opent2t.sample.lamp.superpopular',
                         "translator": "opent2t-translator-com-insteon-lightbulb"
                     };
-                }
-                return undefined;
+                }// else a dimmer switch. 
+                
+                //Since dimmer switch schema is not ready, return light schema.
+                return {
+                    "schema": 'org.opent2t.sample.lamp.superpopular',
+                    "translator": "opent2t-translator-com-insteon-lightbulb"
+                };
```

Comment:
I thought display dimmer switch a an light bulb is just an temporary solution before we get the dimmer switch schema? If we device to have a dimmer switch schema, then we would need to user subCat to differentiate.